src/dxbc/dxbc_options.cpp

```diff
@@ -41,6 +41,7 @@ namespace dxvk {
     enableRtOutputNanFixup   = options.enableRtOutputNanFixup;
     zeroInitWorkgroupMemory  = options.zeroInitWorkgroupMemory;
     forceTgsmBarriers        = options.forceTgsmBarriers;
+    boundsCheckTempStores    = options.boundsCheckTempStores;
```

Comment:
We should enable this unconditionally since it clearly works in D3D11, even though this is an application bug.

Reply:
Done